arch/arm/core/aarch32/cortex_m/fault.c

```diff
@@ -729,6 +730,39 @@ static inline bool z_arm_is_synchronous_svc(z_arch_esf_t *esf)
 	return false;
 }
 
+#if defined(CONFIG_ARMV6_M_ARMV8_M_BASELINE)
+static inline bool z_arm_is_pc_valid(uintptr_t pc)
```

Comment:
Don't we already have something like already @nordic-krch?

Reply:
No, there was similar function to determine if address is in .rodata (for detecting if string is read only).

Reply:
I wonder if it would make sense to make this new `z_arm_is_pc_valid()` a generic one @andyross @dcpleung?